src/argilla/server/apis/v0/handlers/users.py

```diff
@@ -52,7 +56,13 @@ async def whoami(request: Request, current_user: models.User = Security(auth.get
 
     await telemetry.track_login(request, username=current_user.username)
 
-    return User.from_orm(current_user)
+    user = User.from_orm(current_user)
+
+    if current_user.is_admin:
```

Comment:
I think we should add a TODO comment here explaining why this logic is necessary and the conditions when it can be removed.

Reply:
Done